eStore/Core/Networking/eStore.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  eStore.swift
+//  eStore
+//
+//  Created by Vladislav Kondrashkov on 11/17/19.
+//  Copyright © 2019 Vladislav Kondrashkov. All rights reserved.
+//
+
+import Moya
+
+enum eStore {
+    case getSmartphones
```

Comment:
Are you Java man? get? LUL

Reply:
It’s made for differences between request, then I’ll make post and delete ones

Reply:
This is not Swift style at all. Just write `.smartphones` when you wanna get smartphones. Wanna delete them? Say it explicitly. `.deleteSmartphone(id: String)`